tests/tools/nnpackage_run/src/nnpackage_run.cc

```diff
@@ -34,6 +34,10 @@
 #include <unordered_map>
 #include <vector>
 
+#ifdef __clang__
+#include <libgen.h>
```

Comment:
@chunseoklee I've found that there are two version of `basename`. One gets `const char*`, another gets `char*`. Mac OS only supports latter, which requires to include `<libgen.h>`. I've thought using a same version of `basename`, however, it requires to make a copy of string. So. your approach is reasonable.